### PR TITLE
Snowflake test discover

### PR DESCRIPTION
**Pull request by Ismail-Naziz**
## What
Snowflake test discover

Running and passing tests of discover on acceptance tests

Additional: correction of allowed hosts in metadata.yaml

### CODE REVIEW

File: airbyte-integrations/connectors/source-snowflake/metadata.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 data:
   allowedHosts:
     hosts:
-      - ${host#https://}
+      - ${*.snowflakecomputing.com}
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-snowflake/metadata.yaml:4]: I think it should be `- "*. snowflakecomputing.com"`

**Review by maxi297 — APPROVED** [review]
Conditional approval on fixing the allowedHosts

File: airbyte-integrations/connectors/source-snowflake/metadata.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 data:
   allowedHosts:
     hosts:
-      - ${host#https://}
+      - "*. snowflakecomputing.com"
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-snowflake/metadata.yaml:4]: There is an extra space, i believe that should be there. Like this:
- `"*.snowflakecomputing.com"`

**Review by bazarnov — COMMENTED** [review]
LGTM

File: airbyte-integrations/connectors/source-snowflake/metadata.yaml (hunk 2 above)
Comment (bazarnov) [reply airbyte-integrations/connectors/source-snowflake/metadata.yaml:4]: The white-space is not needed, and it's not allowed either for the snowflake domain.

**Review by bazarnov — APPROVED** [review]

**Pull request merged**